@navikt/ds-react/accordion-menu/Item.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { forwardRef, useEffect } from "react";
+import { useStore } from "./ActiveAnchorStore";
+import { OverridableComponent } from "../util";
+import cl from "classnames";
+
+export interface AccordionMenuItemProps {
+  props: {
+    children: React.ReactNode;
+    active?: boolean;
+  } & React.HTMLAttributes<HTMLLIElement>;
+  defaultComponent: "a";
+}
+
+const Item: OverridableComponent<AccordionMenuItemProps> = forwardRef(
```

Comment:
Jeg synes det er forvirrende at denne komponenten er "overridable", men at den ytterste html-taggen den returnerer uansett blir `<li>`. Kan vi f.eks. gjøre så `AccordionMenu` er ansvarlig for å wrappe barna i `<li>`?

Reply:
Ken og jeg se på denne nå.